src/helpers/toast.ts

```diff
@@ -0,0 +1,34 @@
+import { toast as soner } from "vue-sonner"
+
+export type ToastOptions = {
```

Comment:
There are more options that `vue-sonner` provides. For example, description, action and so on. Are we intending to limit the options to only these?